controller/transfer/project_transfer_manager.py

```diff
@@ -148,11 +149,12 @@ def import_file(
     """
     send_progress_update_throttle(project_id, task_id, 0)
     project_item = project.get(project_id)
-    project_item.name = data.get(
-        "project_details_data",
-    ).get(
-        "name",
-    )
+    if not project_item.name:
```

Comment:
In what case would this evaluate to true? (ofc if there is no project name, but I am atm a bit confused when this would be the case)

Reply:
I just saw as soon as I went into the next file, that in the mutation the name is now optional, I was not aware of this

Reply:
Okay, then this comment can be resolved?

Reply:
yes :)